edge-lambdas/src/addons/translations.ts

```diff
@@ -20,6 +20,40 @@ const LANG_QUERY_PARAM = 'lang'
 const LANG_COOKIE_NAME = 'x-pleo-language'
 const TRANSLATION_VERSION_COOKIE_NAME = 'translation-version'
 
+enum SupportedLanguage {
+    DA = 'da',
+    SV = 'sv',
+    EN = 'en',
+    DE = 'de',
+    DE_AT = 'de-AT',
+    ES = 'es',
+    FR = 'fr',
+    FR_BE = 'fr-BE',
+    FI = 'fi',
+    NL = 'nl',
+    NL_BE = 'nl-BE',
+    PT = 'pt',
+    IT = 'it',
+    NO = 'no'
+}
```

Comment:
Let's not use enums. it would be ok just to have an array of of strings in `SUPPORTED_LANGUAGE_LIST` with `as const` (then you can get the union type of you need it)